src/Microsoft.ML.TimeSeries.StaticPipe/TimeSeriesStatic.cs

```diff
@@ -60,11 +58,11 @@ private sealed class Reconciler : EstimatorReconciler
             {
                 Contracts.Assert(toOutput.Length == 1);
                 var outCol = (OutColumn)toOutput[0];
-                return new IidChangePointEstimator(env,
+                return new MLContext().Transforms.IidChangePointEstimator(
```

Comment:
Not sure if we should reuse the `env` passed in.

Reply:
Yea there is no need, it was a miss on my end

Reply:
Can't use env. MLContext implements env. 

---
In reply to: [254475845](https://github.com/dotnet/machinelearning/pull/2344#discussion_r254475845) [](ancestors = 254475845)